ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-answer/activityAnswer.component.ts

```diff
@@ -94,48 +115,83 @@ export class ActivityAnswerComponent {
         this.valueChanged.emit(value);
     }
 
+    ngOnInit(): void {
+        console.log(this.activityGuid, this.studyGuid);
+        console.log(this.isPicklistQuestion(this.block));
+    }
     public isBooleanQuestion(block: AbstractActivityQuestionBlock): boolean {
-        return this.isQuestion(block) && block.questionType === QuestionType.Boolean;
+        return (
```

Comment:
Can you please avoid auto-formatting (at least for everything) ? 
There are lots of format changes in the PR not related to the feature.

Reply:
I think we should apply lint rules when working on the file

Reply:
Hmm. I reverted the change locally (I removed the parentheses and put the whole method body in one line like it was before and `ng lint` said it was fine. So existing code was not violating eslint rules)
@josemanuel167 Could it be that you have different auto-format rules configured in your IDE? I mean, the changes you have here do pass the `ng lint` but they don't really make the code easier to read AND introduce changes to the file that code reviewer need to read and understand that it was just a reformatting, so more work for everyone to figure out what your PR is really doing.
If you don't mind too much, please restore the original formatting. And check why IDE applied this formatting.